spec/Statements.tex

```diff
@@ -238,6 +239,48 @@ \section{The Swap Statement}
 \end{chapel}
 \end{example}
 
+\section{The I/O Statement}
+\label{The_IO_Statement}
+\index{IO!statement}
+\index{statements!IO}
+\index{IO!operator}
+\index{operators!IO}
+
+The I/O operator indicates writing to the left-hand-side the value
```

Comment:
Yes. I updated that sentence. I'm trying to describe the syntax without getting too much into the semantics, because you need to know all about I/O things like Writer for that to make sense...